indexer/src/testnetData.test.ts

```diff
@@ -0,0 +1,166 @@
+import { RpcProvider } from "npm:starknet@5.24.3";
```

Comment:
I think this could all be replaced by a json test file, what do you think? We could keep this script in the repo in case we ever need to regenerate test data but I would prefer to run this once, extract ~500-1000 transactions and dump these in a json.

Reply:
Yes, that makes sense. I will fix it right away. thanks